fastlane/lib/fastlane/actions/zip.rb

```diff
@@ -40,6 +40,16 @@ def self.available_options
         ]
       end
 
+      def self.example_code
+        [
+          "zip",
+          "zip(
```

Comment:
Yeah, different variants, easy example, more advanced one and special case. Whatever is necessary to explain the actoin